tests/foreman/cli/test_contentviews.py

```diff
@@ -142,6 +142,46 @@ def setUp(self):
                 {u'organization-id': TestContentView.org['id']},
                 cached=True)
 
+    def test_bz1184442(self):
```

Comment:
Then I think is better to change the test name. And need to remove `@bz` from the docstring since it is not blocking the test.